pkg/virt-api/webhooks/validating-webhook/validating-webhook_test.go

```diff
@@ -2754,6 +2754,35 @@ var _ = Describe("Function getNumberOfPodInterfaces()", func() {
 		Expect(causes).To(HaveLen(1))
 		Expect(causes[0].Field).To(ContainSubstring("bootOrder"))
 	})
+	It("should reject a serial number whose length is greater than 256", func() {
+		spec := &v1.VirtualMachineInstanceSpec{}
+		sn := strings.Repeat("1", maxStrLen+1)
+
+		spec.Domain.Firmware = &v1.Firmware{Serial: sn}
+
+		causes := ValidateVirtualMachineInstanceSpec(k8sfield.NewPath("fake"), spec)
+		Expect(causes).To(HaveLen(1))
```

Comment:
Please add here the labels. 
`Expect(causes).To(HaveLen(1), "Expect causes length")`
Please do this to the rest of expects.

Reply:
@ksimon1 , I would really prefer to be aligned with the other checks of the length of the causes list here. I do understand your point though, I think that if a label like "Expect causes length" can clarify the current message that is currently produced when these checks fail (Expected <int>: 1 to equal <int>: 2) then we should introduce a function that wraps these check and will set the label rather than passing the very same label everywhere. But I think it is out of the scope of this change.